package.json

```diff
@@ -26,8 +26,6 @@
   ],
   "homepage": "http://harperdb.io",
   "name": "harperdb-connector",
-  "version": "0.0.1",
-  "dependencies": {
-    "harperdb-connect": "^1.2.0"
-  }
+  "version": "1.0.0",
+  "dependencies": {}
 }
```

Comment:
add the following tags with relevant information, I have added few fillers for the same:
- description: "description which shows up here"
![image](https://user-images.githubusercontent.com/12941547/42396890-45c23240-8129-11e8-8bdd-8bc4abff9af7.png)

  "dependencies": {
        "http": "latest"
    },
    "scripts": {
        "jsdoc": "jsdoc2md code/libraries/*/*.js --heading-depth=3 > jsdoc.md"
    },
    "repository": {
        "type": "git",
        "url": "git+https://github.com/rreinold/twilio-sms-library.git"
    },
    "bugs": {
        "url": "https://github.com/rreinold/twilio-sms-library/issues"
    },
    "devDependencies": {
        "jsdoc-to-markdown": "^4.0.1"
    }